### PR TITLE
feat: govdao mvp improvement

### PR DESCRIPTION
**Pull request by moul**
Continues #1945

### CODE REVIEW

File: examples/gno.land/r/gov/dao/dao.gno [hunk 1]
```diff
@@ -2,27 +2,44 @@ package govdao
...
+	accepted bool
+	finished bool
+}
+
+func (p Proposal) Status() string {
+	if p.Executor.Done() {
+		return p.Executor.Status()
+	}
+	if p.accepted {
+		return "accepted"
```
Comment (zivkovicmilos) [inline examples/gno.land/r/gov/dao/dao.gno:31]: What do you think about defining a type alias for these states?

ex:

```go
type Status string

var (
    Accepted Status = "accepted"
    // ...
)
```

File: examples/gno.land/r/gov/dao/dao.gno [hunk 2]
```diff
@@ -32,34 +49,81 @@ func Propose(proposal Proposal) int {
...
+		return output
+	}
+
+	// else display the proposal
+	idx, err := strconv.Atoi(path)
+	if err != nil {
+		return "404"
+	}
+
+	println("AAA", idx)
```
Comment (zivkovicmilos) [inline examples/gno.land/r/gov/dao/dao.gno:103]: Leftover?

File: examples/gno.land/p/gov/proposal/proposal.gno [hunk 3]
```diff
@@ -1,6 +1,10 @@
 // Package proposal provides a structure for executing proposals.
 package proposal
 
+import "std"
+
+const daoPkgPath = "gno.land/r/gov/dao" // XXX: make it configurable with r/sys/vars?
```
Comment (zivkovicmilos) [inline examples/gno.land/p/gov/proposal/proposal.gno:6]: I like this idea 👌

**Review by zivkovicmilos — APPROVED** [review]
Looks good 💯 

I've left a few nitpicks, otherwise good to go 🚀

File: examples/gno.land/r/gov/dao/dao.gno (hunk 2 above)
Comment (moul) [reply examples/gno.land/r/gov/dao/dao.gno]: fixed

File: examples/gno.land/r/gov/dao/dao.gno (hunk 1 above)
Comment (moul) [reply examples/gno.land/r/gov/dao/dao.gno:31]: oh yes 👍

Comment (moul) [reply examples/gno.land/r/gov/dao/dao.gno]: fixed in 8ecee2f

**Review by gfanton — APPROVED** [review]

**Pull request merged**